pkg/keda/kedacontroller.go

```diff
@@ -0,0 +1,310 @@
+package keda
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	kedav1alpha1 "github.com/kedacore/keda-olm-operator/apis/keda/v1alpha1"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	goclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// KedaControllerBuilder provides a struct for KedaController object from the cluster and a KedaController definition.
+type KedaControllerBuilder struct {
```

Comment:
You have "keda" in package name. Please remove keda from structs and functions names.
For example:

- type ControllerBuilder
- func NewControllerBuilder
- func PullKController
...

Same for test function names

Reply:
same package keda will include scaleObject and triggerAuthentication and maybe more objects will be neede, I think it will be better to leave like this (with a full obj name) 
this is api object name = "KedaController"

Reply:
@kononovn there are many different types of object related to the keda operator and all have a different name, not always starts from "keda" 
and keda is an operator name

Reply:
@elenagerman It's go best practice. 
The point is.
Indead of 
kede. NewKedaControllerBuilder

simply use:
keda. NewControllerBuilder

As a result we don't use keda multiple times and at the same time we undertand that Controller is related to keda only.